src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -225,8 +222,9 @@
     private final String coreLoadPath;
 
     @TruffleBoundary
-    private static Source initCoreSource() {
-        return Source.newBuilder("").name("(core)").mimeType(RubyLanguage.MIME_TYPE).build();
+    private static SourceSection initCoreSourceSection() {
+        final Source source = Source.newBuilder("").name("(core)").mimeType(RubyLanguage.MIME_TYPE).build();
```

Comment:
Is this empty string not going to be confusing?

Reply:
It was already there. What do you we should change it to?

Reply:
Oh right yes I see it was already there.